README.md

```diff
@@ -13,6 +13,7 @@ This action will bump version, tag commit and generate a changelog with conventi
 - **Optional** `git-branch`: The branch used to push. Default is the current branch (`${{ github.ref }}`)
 - **Optional** `git-url`: Git repository domain. Default is `github.com`
 - **Optional** `git-path`: Path filter for the logs. If set, only commits that match the path filter will be considered. By default, we won't use this feature(empty string).
+- **Optional** `infile`: Read the CHANGELOG from this file.
```

Comment:
Could we bring this one inline with `output-file`? So `input-file`, maybe also describe a bit more what it does? If I understood correct :`Read the CHANGELOG from this file and prepend the new release to it`.